extension/syntaxes/dvcYaml.tmGrammar.json

```diff
@@ -0,0 +1,50 @@
+{
+  "scopeName": "source.yaml",
```

Comment:
This was an experiment to add highlighting to specific dvc.yaml keywords, but it needs improvement/thinking/discussion so I'll remove it for this PR too